Test/git-issues/git-issue-4048.dfy

```diff
@@ -0,0 +1,6 @@
+// RUN: %exits-with 1 %baredafny build -t:lib "%s" x.doo > "%t"
```

Comment:
You should do `2>` instead so that the diff can actually check the stderr output.

Reply:
Fixed